apps.go

```diff
@@ -168,23 +168,13 @@ func (s *appsService) AppsEnsureRepo(app *App, repo string) error {
 // AppsFindOrCreateByRepo first attempts to find an app by repo, falling back to
 // creating a new app.
 func (s *appsService) AppsFindOrCreateByRepo(repo string) (*App, error) {
-	a, err := s.store.AppsFirst(AppsQuery{Repo: &repo})
-	if err != nil && err != gorm.RecordNotFound {
-		return a, err
-	}
-
-	// If the app wasn't found, create a new app linked to this repo.
-	if err != gorm.RecordNotFound {
-		return a, nil
-	}
-
-	n := NewAppNameFromRepo(repo)
-
-	a, err = s.store.AppsFirst(AppsQuery{Name: &n})
+	n := AppNameFromRepo(repo)
+	a, err := s.store.AppsFirst(AppsQuery{Name: &n})
```

Comment:
I think if we make this change, then we're actually not using the `repo` column anywhere and it could just be removed (doesn't have to happen in this PR).

I was originally thinking that the CLI would allow you to link a docker repo to an app, but I think I might like this approach better.